### ISSUE TITLE
Build failure - HighlightedText.tsx

### ISSUE THREAD

**Issue by g3rg** [Floppy/dataslate#606]
After the latest PR merges, I'm getting the error below, haven't had a chance to look closely at it, but thought I'd raise an issue in case someone else is looking at it.

Type 'ReactNodeArray' is not assignable to type 'ReactNode[]'.  TS2322

     8 | export function HighlightedText (props: Props) {
     9 |   let output = [props.children]
  > 10 |   output = reactStringReplace(output, '▲', () => <span className='triangle'>▲</span>)
       |   ^
    11 |   output = reactStringReplace(output, '●', () => <span className='circle'>●</span>)
    12 |   output = reactStringReplace(output, '○', () => <span className='circle'>●</span>)
    13 |   output = reactStringReplace(output, '⬤', () => <span className='circle'>●</span>)

**Comment by g3rg** [comment]
@Floppy @lstrzebinczyk I won't get a chance to look at this till probably Tuesday, but just wondering if either of you have this issue? I can't build/ run the floppy/main branch because of it.

**Comment by Floppy** [comment]
Yes, sorry, meant to reply. I have it replicated here too; looking into it at the moment, trying to work out what the hell is wrong with the type mismatch. I've also added the thing that caught it to the CI build so we won't accidentally get build breakages in future without realising.
